sig/deepin-surface/metadata.yml

```diff
@@ -2,15 +2,27 @@
 name: deepin-surface
 blog:
 rss:
+matrix: '#deepin-surface:mozilla.org'
 proposal:
   issue: 10
   by:
     handle: zccrs
     id: 13449038
 date-created: '2022/08/10'
 date-archived: '-'
-team:
+team: sig-deepin-surface
 repos:
   maintained:
+    - sig-deepin-surface
   package:
 members:
+  - handle: shiptux
+    id: 22735550
+  - handle: iamhyc
+    id: 9068301
+  - handle: xiekehui
+    id:
```

Comment:
```suggestion
    id: 20040331
```

Reply:
Is it the same person on deepin forum?

Reply:
If it isn't or if you are not sure about it, then it shouldn't be listed here.

If you would like to list a member that doesn't have GitHub account, you can edit MEMBERS.md to add their information manually.

Reply:
All the members have already been recorded in `MEMBERS.md` file.

If this metadata file is supposed to only record Github accounts, I can remove the entry for "xiekehui".

Reply:
> If this metadata file is supposed to only record Github accounts, I can remove the entry for "xiekehui".

It is only for GitHub accounts currently. Please remove or comment out that item for now and maybe we can improve the yaml support in the future :)